src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2096,7 +2096,7 @@ public static string NameString(PSObject instance)
             {
                 return instance?.BaseObject is FileSystemInfo fileInfo
                     ? InternalSymbolicLinkLinkCodeMethods.IsReparsePointLikeSymlink(fileInfo)
-                        ? $"{fileInfo.Name} -> {InternalSymbolicLinkLinkCodeMethods.GetTarget(instance)}"
+                        ? $"{fileInfo.Name} -> {fileInfo.LinkTarget}"
```

Comment:
Design intention of the new LinkTarget() API was to rid of IsReparsePointLikeSymlink() :-)

(I don't ask to do this in the PR but we could review uses of IsReparsePointLikeSymlink())

Reply:
By reviewing the code, it looks to me we can replace `IsReparsePointLikeSymlink(fileInfo)` with `fileInfo.LinkTarget` only if we don't treat `AppExecLink` as a symlink-like reparse point. But let's defer the discussion to it's own PR/issue.

Reply:
So we wait public API for AppExecLink?

Reply:
That could be an option. We will have conversation with the Windows App team again, so if they agreed to document the API, .NET can support AppExecLink, and we can get it for free. But for now, I'd say keep `IsReparsePointLikeSymlink`.

Reply:
If AppExecLink will be public this can be addressed in `fileInfo.LinkTarget`. I think this was a design intention in the discussion with .Net Library team as well as removing `IsReparsePointLikeSymlink()`.

Reply:
All that will have to wait until the AppExceLink API/struct is officially documented.